src/game/SpellEffects.cpp

```diff
@@ -4894,7 +4940,14 @@ void Spell::EffectDurabilityDamage(SpellEffectIndex eff_idx)
         return;
 
     if (Item* item = ((Player*)unitTarget)->GetItemByPos(INVENTORY_SLOT_BAG_0, slot))
+    {
         ((Player*)unitTarget)->DurabilityPointsLoss(item, damage);
+
+        ExecuteLogInfo info(unitTarget->GetObjectGuid());
+        info.durabilityDamage.itemEntry = item->GetProto()->ItemId;
+        info.durabilityDamage.unk = -1;
```

Comment:
It is. Take a closer to look to a structure here: https://gist.github.com/Laizerox/d32f4d57e2f8304d047bcf2713903e32#file-spelllogexecute-cpp-L195 and here https://gist.github.com/Laizerox/d32f4d57e2f8304d047bcf2713903e32#file-spelllogexecute-cpp-L196 and here https://gist.github.com/Laizerox/d32f4d57e2f8304d047bcf2713903e32#file-spelllogexecute-cpp-L200